package.json

```diff
@@ -92,38 +93,47 @@
     "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
+    "@emotion/core": "^10.0.16",
+    "@svgr/webpack": "^4.3.2",
+    "@testing-library/jest-dom": "^4.0.1",
     "autoprefixer": "^9.6.1",
     "axios": "^0.19.0",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^3.1.0",
     "dotenv-webpack": "^1.7.0",
     "express": "^4.17.1",
+    "formik": "^1.5.8",
     "glob": "^7.1.4",
     "html-webpack-plugin": "^3.2.0",
+    "i": "^0.3.6",
     "jest-svg-transformer": "^1.0.0",
     "jwt-decode": "^2.2.0",
     "mini-css-extract-plugin": "^0.8.0",
+    "npm": "^6.11.0",
```

Comment:
kindly remove this as it must have been installed in err.

Reply:
👍